Library/Formula/llvm.rb

```diff
@@ -117,26 +116,19 @@ def install
 
     ENV["REQUIRES_RTTI"] = "1" if build.with?("rtti") || build.with?("clang")
 
-    args = [
-      "--prefix=#{prefix}",
-      "--enable-optimized",
-      # As of LLVM 3.1, attempting to build ocaml bindings with Homebrew's
-      # OCaml 3.12.1 results in errors.
-      "--disable-bindings",
+    args = %w[
+      -DLLVM_OPTIMIZED_TABLEGEN=On
     ]
 
-    if build.with? "all-targets"
-      args << "--enable-targets=all"
-    else
-      args << "--enable-targets=host"
-    end
-    args << "--enable-shared" if build.with? "shared"
+    args << "-DBUILD_SHARED_LIBS=Off" if build.without? "shared"
 
-    args << "--disable-assertions" if build.without? "assertions"
+    args << "-DLLVM_ENABLE_ASSERTIONS=On" if build.with? "assertions"
 
-    system "./configure", *args
-    system "make"
-    system "make", "install"
+    mktemp do
+      system "cmake", "-G", "Unix Makefiles", buildpath, *std_cmake_args, *args
```

Comment:
`*foo, *bar` doesn't work in Ruby 1.8 (on Mountain Lion); use `*(foo+bar)` or similar.